test/torchaudio_unittest/transforms/batch_consistency_test.py

```diff
@@ -33,13 +33,14 @@ def test_batch_Resample(self):
         self.assertEqual(computed, expected)
 
     def test_batch_MelScale(self):
-        specgram = torch.randn(2, 31, 2786)
+        n_stft = 31
+        specgram = torch.randn(2, n_stft, 2786)
```

Comment:
Can you use the default value of `n_stft=201`, so that we can catch an error for default value first if that happens in the future?
Also not sure where the 2786 is coming from, but I think we can use something shorter, like 256.